lib/console/jobs.ex

```diff
@@ -91,7 +91,7 @@ defmodule Console.Jobs do
 
   def delete_sent_alerts do
     # since events are kept as "sent" so we can check against flapping, delete them in 24-hr batches
```

Comment:
can you update the comment too so we don't think the 25 is a typo lol

Reply:
good point!